app/src/citations.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import random
+import re
+from typing import Match, Sequence
+
+from src.db.models.document import Chunk, ChunkWithScore
+from src.format import _get_bem_url
+
+logger = logging.getLogger(__name__)
+
+_footnote_id = random.randint(0, 1000000)
+_footnote_index = 0
+
+
+def get_context_for_prompt(chunks: Sequence[ChunkWithScore]) -> str:
+    return "\n\n".join(
+        [
+            f"Citation: chunk-{index} \nContent: {chunk.chunk.content}"
```

Comment:
Let's keep the document name since it might be useful for the LLM too
```suggestion
            f"Citation: chunk-{index}\nDocument name: {chunk.document.name}\nContent: {chunk.chunk.content}"
```